### PR TITLE
Add usage info to command missing them

### PR DESCRIPTION
**Pull request by yiwen101**
Add usage info to delete task and link command.
Replace person list with student list, lesson list with schedule list.
Change to "any number of unique flag".

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddLessonCommandParser.java [hunk 1]
```diff
@@ -50,9 +50,9 @@ public static Lesson parseLesson(String args, boolean nameIsOptional) throws Par
         }
     }
     private static String getUsageInfo() {
-        return "\nUsage: addLesson -name [NAME] (any number of -[subject|day|start|end|remark] [value]). "
-                + "\n For example, addLesson -name John -subject English -day 23 -start 14:30 -end 16:30"
-                + "\n If you are currently displaying lesson list, you could use 'add' inplace of 'addLesson'. "
+        return "\nUsage: addLesson -name [NAME] (any number of unique -[subject|day|start|end|remark] [value]). "
+                + "\n For example, addLesson -name John -subject Math -day 23 -start 14:30 -end 16:30"
```
Comment (richiehx) [inline src/main/java/seedu/address/logic/parser/AddLessonCommandParser.java]: I think Math has to be Mathematics in order for the subject to be valid.

File: src/main/java/seedu/address/logic/parser/TypeParsingUtil.java [hunk 2]
```diff
@@ -118,7 +118,8 @@ public static LocalDate parseDate(String input) throws ParseException {
             int day = TypeParsingUtil.parseNum(dayM.group(1), 1, findMaxDay(now.getYear(), now.getMonthValue()));
             return LocalDate.of(now.getYear(), now.getMonth(), day);
         } else {
-            throw new InvalidInputException(input + " is not a valid date");
+            throw new InvalidInputException(input
+                    + " is not a valid date, please use yyyy/mm/dd or mm/dd or dd");
```
Comment (HollaG) [inline src/main/java/seedu/address/logic/parser/TypeParsingUtil.java]: Might be good to have an example here

**Review by HollaG — COMMENTED** [review]
LGTM just that one minor comment thing

**Review by richiehx — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/parser/TypeParsingUtil.java (hunk 2 above)
Comment (yiwen101) [reply src/main/java/seedu/address/logic/parser/TypeParsingUtil.java]: sure

Comment (yiwen101) [reply src/main/java/seedu/address/logic/parser/TypeParsingUtil.java]: Fixed

File: src/main/java/seedu/address/logic/parser/AddLessonCommandParser.java (hunk 1 above)
Comment (yiwen101) [reply src/main/java/seedu/address/logic/parser/AddLessonCommandParser.java]: Thanks.
Fixed

**Pull request merged**